ArmaforcesMissionBot/Helpers/SignupHelper.cs

```diff
@@ -172,6 +172,8 @@ public static async Task<EmbedBuilder> CreateMainEmbed(SocketGuild guild, Missio
                     .AddField("Zamknięcie zapisów:", mission.CloseTime.ToString())
                     .WithAuthor(guild.GetUser(mission.Owner));
 
+            mainEmbed.Author.Url = "https://discord.com/users/" + (mission.Owner).ToString();
```

Comment:
Consider extracting `"https://discord.com/users/"` to `DiscordConstants` like I proposed already. Having hardcoded constants can easily lead to multiple usages which are not forced to be equal (thus are prone to errors and hard to change).